assets/src/edit-story/app/media/media3p/categories/actions.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import * as types from './types';
+
+export const fetchCategoriesStart = (dispatch) => ({ provider } = {}) => {
+  dispatch({
+    type: types.FETCH_MEDIA_CATEGORIES_START,
+    payload: {
+      provider: provider,
+    },
+  });
+};
+
+export const fetchCategoriesSuccess = (dispatch) => ({
+  provider,
+  ...otherProperties
```

Comment:
[nit]: The properties can be explicit here. '...otherProperties' exist in the media pagination logic only so that 'local' can add their own properties.

Reply:
Done.